src/neoTrackerPanel.ts

```diff
@@ -165,9 +167,33 @@ export class NeoTrackerPanel implements INeoSubscription, INeoStatusReceiver {
                     ((this.viewState.currentBlock !== undefined) ? ActivePage.BlockDetail : ActivePage.Blocks );
             } else if (message.e === trackerEvents.Copy) {
                 await vscode.env.clipboard.writeText(message.c);
+            } else if (message.e === trackerEvents.Search) {
+                const asNumber = parseInt(message.c);
+                const block = await this.rpcConnection.getBlock((asNumber + '') === message.c ? asNumber : message.c, this);
+                if (block) {
+                    this.viewState.currentBlock = block;
+                    this.viewState.activePage = ActivePage.BlockDetail;
+                } else {
+                    const transaction = await this.rpcConnection.getTransaction(message.c, this);
```

Comment:
if parseInt works, but getBlock fails, do we also need to call getTx or check to see if it's an address?

Reply:
checking if it is a tx: technically yes, but practically no (a hash that by coincidence contains no a-f digits could be a block number, but way too big to correspond to a block in practice)

checking if it is an address: no (addresses never overlap with block numbers because they always begin with a letter)

Reply:
technically there is an edge case where a tx hash may contain no a-f digits (albeit a bit unlikely) so we should still try getTx

I'm going to move the check for address format higher up per your other comment, so that part becomes irrelevant now.